src/services/doppler-api-client.double.ts

```diff
@@ -95,6 +95,24 @@ const subscriber = {
   unsubscriptionComment: 'test',
   status: 'unsubscribed_by_hard',
   score: 2,
+  _links: [
+    {
+      href: '/accounts/test@test.com',
+      description: 'Get account home.',
+      rel: '/docs/rels/get-account-home',
+    },
+    {
+      href:
+        '/accounts/test@test.com/subscribers/test@test.com/permissions-history.csv?_s=XpYGFksgzI6LhSXu2T4LZLlYUHzr6GN68QWvw19m1yw6LdrNklO5qpwARR4WEJVK',
+      description: "Get subscriber's complete permission history as CSV",
+      rel: '/docs/rels/get-permission-history-csv',
+    },
+    {
+      href: '/accounts/test@test.com/lists/27766804/subscribers/test@test.com',
+      description: 'Remove Subscriber from List.',
+      rel: '/docs/rels/remove-from-list',
+    },
+  ],
```

Comment:
I do not see the related changes in [doppler-api-client](https://github.com/FromDoppler/doppler-webapp/blob/master/src/services/doppler-api-client.ts).

So, it will not work with real data, does not it?